android/app/src/main/res/layout/launch_screen.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@usaidpeerzada Is this file needed and being used anywhere?

Reply:
Yes, this is the file  react-native-splash-screen package reads  and this file needs to be exactly as named and in layout folder to work properly.